src/hotspot/share/runtime/os.cpp

```diff
@@ -1812,6 +1812,9 @@ char* os::reserve_memory(size_t bytes, bool executable, MEMFLAGS flags) {
   char* result = pd_reserve_memory(bytes, executable);
   if (result != nullptr) {
     MemTracker::record_virtual_memory_reserve(result, bytes, CALLER_PC, flags);
+    log_debug(os,map)("Reserved [" INTPTR_FORMAT " - " INTPTR_FORMAT"] (%zu bytes).", p2i(result), p2i(result + bytes), bytes);
```

Comment:
Space needed after INTPTR_FORMAT in `INTPTR_FORMAT"` - this applies to multiple locations